src/imitation/scripts/common/reward.py

```diff
@@ -31,18 +32,30 @@ def normalize_input_running():
     net_kwargs = {"normalize_input_layer": networks.RunningNorm}  # noqa: F841
 
 
+@reward_ingredient.named_config
+def normalize_output_disable():
+    net_kwargs = {"rew_normalize_class": nn.Identity}  # noqa: F841
+
+
+@reward_ingredient.named_config
+def normalize_output_running():
+    net_kwargs = {"rew_normalize_class": networks.RunningNorm}  # noqa: F841
+
+
 @reward_ingredient.config_hook
 def config_hook(config, command_name, logger):
     """Sets default values for `net_cls` and `net_kwargs`."""
     del logger
     res = {}
     if config["reward"]["net_cls"] is None:
-        default_net = reward_nets.BasicRewardNet
+        default_net = reward_nets.BasicNormalizedRewardNet
         if command_name == "airl":
             default_net = reward_nets.BasicShapedRewardNet
         res["net_cls"] = default_net
     if "normalize_input_layer" not in config["reward"]["net_kwargs"]:
         res["net_kwargs"] = {"normalize_input_layer": networks.RunningNorm}
+    if "rew_normalize_class" not in config["reward"]["net_kwargs"]:
+        res["net_kwargs"] = {"rew_normalize_class": networks.RunningNorm}
     return res
 
 
```

Comment:
To elaborate a bit on what I mentioned in an earlier comment, I was imagining on adding something like an `output_normalize_class` config parameter (defaulting to None), modifying the below `make_reward_net` function to take that as a parameter and then if non-None wrapping the constructed reward network with the `NormalizedRewardNet` class before returning.

This seems cleaner, and right now I think you can't have a normalized shaped reward network for example, which is kind of unfortunate.

Reply:
Yeah, I think this makes sense.